plugins/SliceInfoPlugin/SliceInfo.py

```diff
@@ -112,9 +114,12 @@ def _onWriteStarted(self, output_device):
 
             # Submit data
             try:
-                f = urllib.request.urlopen(self.info_url, data = binary_data, timeout = 1)
-                Logger.log("i", "Sent anonymous slice info to %s", self.info_url)
-                f.close()
+                if Platform.isOSX():
+                    ssl_context = ssl._create_unverified_context()
+                    server_request = urllib.request.urlopen(self.info_url, data = binary_data, timeout = 1, context = ssl_context)
+                else:
+                    server_request = urllib.request.urlopen(self.info_url, data = binary_data, timeout = 1)                Logger.log("i", "Sent anonymous slice info to %s", self.info_url)
```

Comment:
That's not the problem; scroll to the right and you'll find a Logger statement that should be on the next line.